### PR TITLE
Add plugin: Sync Vault CE

### PR DESCRIPTION
**Pull request by abcamus**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin:

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by joethei** [comment]
[# 🌟 Obsidian Sync Vault CE (Community Edition)](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/README.md#L3), [# 🌟 Obsidian Sync Vault 社区版](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/README.zh_cn.md#L3), [Obsidian Sync Vault CE](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/README.md#L7), [Obsidian Sync Vault](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/README.zh_cn.md#L7)
Naming something "Obsidian xyz" is reserved for first party products we create, please change this header.

["fundingUrl": "https://kqiu.top/welcome-to-sync-vault/",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/manifest.json#L9)
This is meant for links to services like _Buy me a coffee_, _GitHub sponsors_ and so on, if you don't have such a link remove this line.

[leaves.forEach(leaf => leaf.detach());](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/main.ts#L143), [leaves.forEach(leaf => leaf.detach());](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/main.ts#L149), [await this.openView(workspace);](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/main.ts#L157), [leaves.forEach(leaf => leaf.detach());](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/main.ts#L164)
Do not detach leaves with your custom view here, this is an antipattern, see: <https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Don't+detach+leaves+in+%60onunload%60>
Instead, only create a new leaf, if there is not one already.
If there is a leaf, focus on it.

[export async function delay(ms: number): Promise<void> {](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/util/time.ts#L1-L3)
There is `sleep` in the Obsidian API already, which does the same thing.

[export const getSyncButtonText = (status: SyncStatus): string => {](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/model/sync-status.tsx#L83-L107)
This shouls also be translatable.

["name": "Sync Vault",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L7-L11), ["rootFolder": "Root Folder",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L21), ["vaultDesc": "Cloud Path: %{syncPath}",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L26), ["title": "Get Started",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L30), ["title": "Sync Settings",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L34), ["selfControlled": "Self Controlled",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L37), ["title": "Upload Mode",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L40), ["userControl": "Controlled Upload",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L42), ["title": "Download Mode",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L46), ["autoOnLoad": "Update on Load",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L48-L49), ["title": "Select Cloud Storage",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L53), ["title": "Access Token",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L62), ["clickToAuthorize": "Click to Authorize",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L64), ["title": "Auto Sync on Load",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L69), ["title": "Log Mode",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L73), ["openLogFile": "View Logs",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L75)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

["title": "Sync Settings",](https://github.com/abcamus/obsidian-sync-vault-ce/blob/643ffa31215fedcad59503e2146ffa1b2790d2e5/src/i18n/locales/en.ts#L34) 
[Avoid "settings" in settings headings](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Avoid+%22settings%22+in+settings+headings)

**Comment by abcamus** [comment]
Thanks for your review. Changes have been made at: [new commit](https://github.com/abcamus/obsidian-sync-vault-ce/commit/001cb5596f13969c9a3355d3eadaf34beee60dc1)

Updates as follows:

1. rename "Obsidian Sync Vault CE" to "Sync Vault CE".
2. remove logic of manually closing custom view.
3. update translations in `i18n/zh.ts`.
4. remove `delay` function in `utils.time` module.

**Comment by joethei** [comment]
[en.ts](https://github.com/abcamus/obsidian-sync-vault-ce/blob/001cb5596f13969c9a3355d3eadaf34beee60dc1/src/i18n/locales/en.ts#L1-L96)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

**Comment by abcamus** [comment]
new commit: [update en.ts to be sentence style](https://github.com/abcamus/obsidian-sync-vault-ce/commit/cf9b4743ac024116adc6080496d9e432c06759f5)
